syft/pkg/cataloger/sbom/cataloger.go

```diff
@@ -0,0 +1,71 @@
+package sbom
+
+import (
+    "bytes"
+    "fmt"
+    "github.com/anchore/syft/internal/formats/cyclonedxjson"
+    "github.com/anchore/syft/internal/formats/cyclonedxxml"
+    "github.com/anchore/syft/internal/formats/spdx22json"
+    "github.com/anchore/syft/internal/formats/spdx22tagvalue"
+    "github.com/anchore/syft/internal/formats/syftjson"
+    "github.com/anchore/syft/syft/artifact"
+    "github.com/anchore/syft/syft/pkg"
+    "github.com/anchore/syft/syft/pkg/cataloger/common"
+    "github.com/anchore/syft/syft/sbom"
+    "io"
+)
+
+// NewSBOMCataloger returns a new SBOM cataloger object loaded from saved SBOM JSON.
+func NewSBOMCataloger() *common.GenericCataloger {
+    globParsers := map[string]common.ParserFn{
+        "**/syft.json":   parseSyftJson,
```

Comment:
I believe this is not a valid syft file pattern. The IANA registered type for syft is *.syft.json per https://www.iana.org/assignments/media-types/application/vnd.syft+json

Reply:
cleaned up